MerlinAU.sh

```diff
@@ -2204,56 +2204,66 @@ _GetLoginCredentials_()
     return 0
 }
 
-##---------------------------------------##
-## Added by ExtremeFiretop [2024-Mar-26] ##
-##---------------------------------------##
-_PermNodeList_() {
-    # Get the value of asus_device_list
+##----------------------------------------##
+## Modified by Martinski W. [2024-Apr-06] ##
+##----------------------------------------##
+_GetNodeIPv4List_()
+{
+    # Get the value of asus_device_list #
+    local ip_addresses
     local device_list="$(nvram get asus_device_list)"
     local urlDomain="$(nvram get lan_ipaddr)"
 
-    # Check if asus_device_list is not empty
-    if [ -n "$device_list" ]; then
-        # Split the device list into records and extract the IP addresses, excluding the lan_ipaddr
-        local ip_addresses=$(echo "$device_list" | tr '<' '\n' | awk -v exclude="$urlDomain" -F'>' '{if (NF>=4 && $3 != exclude) print $3}')
+    # Check if asus_device_list is not empty #
+    if [ -n "$device_list" ]
+    then
+        # Split the device list into records and extract the IP addresses, excluding the lan_ipaddr #
+        ip_addresses="$(echo "$device_list" | tr '<' '\n' | awk -v exclude="$urlDomain" -F'>' '{if (NF>=4 && $3 != exclude) print $3}')"
 
-        # Check if IP addresses are not empty
+        # Check if IP addresses are not empty #
         if [ -n "$ip_addresses" ]; then
             # Print each IP address on a separate line
             printf "%s\n" "$ip_addresses"
         else
-            echo "Error: Unable to extract IP addresses from asus_device_list."
+            echo "**ERROR**: Unable to extract IP addresses from asus_device_list."
             return 1
         fi
     else
-        Say "asus_device_list is not populated. Zero Nodes Found."
+        Say "NVRAM asus_device_list is NOT populated. No Mesh Nodes were found."
+        return 1
     fi
+    return 0
 }
 
-##---------------------------------------##
-## Added by ExtremeFiretop [2024-Mar-27] ##
-##---------------------------------------##
-_NodeActiveStatus_() {
-    # Get the value of cfg_device_list
+##----------------------------------------##
+## Modified by Martinski W. [2024-Apr-06] ##
+##----------------------------------------##
+_NodeActiveStatus_()
```

Comment:
Same with this, approved formatting improvements, may still go back to remove the echo error messages.